aspnetcore/performance/response-compression/sample/ResponseCompressionSample.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">

```

Comment:
I'd say take a look at https://github.com/aspnet/Security/tree/dev/samples for a few samples we have converted to .csproj for guidance on what it should look like.

@natemcmaster and @BrennanConroy will probably have more context on updating to vs2017.